doc/development.md

```diff
@@ -114,11 +110,11 @@ Create a new [release checklist](https://github.com/coreos/ignition/issues/new?l
 
 ## Marking an experimental spec as stable
 
-When an experimental version of the Ignition config spec (e.g.: `2.3.0-experimental`) is to be declared stable (e.g. `2.3.0`), there are a handful of changes that must be made to the code base. These changes should have the following effects:
+When an experimental version of the Ignition config spec (e.g.: `3.1.0-experimental`) is to be declared stable (e.g. `3.1.0`), there are a handful of changes that must be made to the code base. These changes should have the following effects:
```

Comment:
Should we switch to generics for version numbers (e.x. `X.Y.0-experimental`) here like we do in other sections?

Reply:
I think this is OK. This section is an overview while the following sections are specific instructions.

Reply:
My line of thinking was that we wouldn't need to update the text again if we ended up making another major version bump in the future. Either way I don't think it's a major issue either way and am ok with whichever you choose.